<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2121</width>
    <height>949</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="imagen" native="true">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>40</y>
      <width>411</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Nombre">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Introduce el nombre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DNI">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>250</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Introduce el DNI</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Apellido1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Introduce el primer apellido</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Apellido2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>400</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Introduce el segundo apellido</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Correo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>550</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Introduce el correo electrónico</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Telefono">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>550</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Introduce el telefono</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Direccion">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>680</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Introduce la dirección</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>780</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registrar cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVolver">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>780</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
